core/src/main/java/com/linecorp/armeria/server/file/DefaultMediaTypeResolver.java

```diff
@@ -114,5 +108,12 @@ static MediaType guessFromPath(String path, @Nullable String contentEncoding) {
         return guessFromPath(path.substring(0, path.lastIndexOf('.')));
     }
 
-    private MimeTypeUtil() {}
+    private static void add(Map<String, MediaType> extensionToMediaType,
```

Comment:
Hi @jrhee17👋. Should we categorize `private static void add` method as `utility methods (both static and member)` or `static methods` as mentioned in Armeriadeveloper-guide?

Reply:
Since `add` method is used to build `EXTENSION_TO_MEDIA_TYPE`, we can think of `add()` as a static method.
If it doesn't hurt readability, it might be better to group together.
But sometimes, in a big class, it is convenient to place a private static method nearby a call site.